k8s-traffic-logger.py

```diff
@@ -166,6 +189,13 @@ def get_ipv6_key(k):
 
     print()
 
+    # Get pod metadata
+    all_pod_metadata = defaultdict(lambda: ["",""])
+    ret = v1.list_pod_for_all_namespaces(watch=False)
```

Comment:
This downloads the full manifests for all pods in the cluster every 10 seconds. You should limit it to the pods on this node at least, possibly getting only the IP address field (with [table option](https://kubernetes.io/docs/reference/using-api/api-concepts/#receiving-resources-as-tables) maybe?), possibly watch for changes

Reply:
added `field_selector` but was not able to find API table option for this method in the [python API doc](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CoreV1Api.md#list_pod_for_all_namespaces). If we wanna use this, then we'll need to use HTTP to get manifest instead of client lib.